### DISCUSSION TITLE
Horizontal scroll based on mouse wheel

### DISCUSSION THREAD

**Discussion by stspapa95** [Norserium/react-indiana-drag-scroll#70] [General]
Is there any way to add/implement horizontal scroll based on mouse wheel by keeping the existing drag functionality?

Thank you in advance!!

**Comment by Norserium** [comment]
@stspapa95, have you ever seen the examples of horizontal scroll by a mouse wheel? As a workaround, you can process `onWheel` event (currently only for the parent container) and scroll the container programmatically.

**Comment by Norserium** [comment]
It should work in [3.x.x](https://norserium.github.io/react-indiana-drag-scroll/). Try to update to the newest version:
```bash
npm install react-indiana-drag-scroll@next
```
